python/tvm/meta_schedule/relax_integration.py

```diff
@@ -193,6 +194,8 @@ def tune_relax(
         The maximum number of trials to run
     max_trials_per_task : Optional[int]
         The maximum number of trials to run for each task
+    op_names: Optional[List[str]]
+        A list of operator names to specify which op to tune
```

Comment:
Maybe we can mention that by default is None and means all op would be tuned.

Reply:
Done